### ISSUE TITLE
not able to get into swagger / redoc

### ISSUE THREAD

**Issue by gironaptik** [retspen/webvirtcloud#560]
Hi,

Trying to get rest list via swagger / redoc and get the following error:
Not Found
The requested resource was not found on this server.

Something that need to be installed / configured before?

Thanks

**Comment by catborise** [comment]
is your installation new? did you check required packages? `pip3 install -r conf/requirements.txt`

**Comment by gironaptik** [comment]
Yes, perform new installation and checked the requirement -  its all updated
seems like it missing index page

**Comment by catborise** [comment]
i tested it again with docker installation, it works as expected. 
can you please edit settings.py file to set debug true. may be we can see the problem.
